app/assets/stylesheets/active_admin/components/_dropdown_menu.scss

```diff
@@ -1,151 +1,56 @@
-.dropdown_menu {
-  display: inline;
-
-  .dropdown_menu_button {
-    @include light-button;
-    position: relative;
-    padding-right: 22px !important;
-    cursor: pointer;
-
-    &:before {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #FFF transparent;
-      right: 12px;
-      top: 45%;
-    }
-
-    &:after {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #777 transparent;
-      right: 12px;
-      top: 45%;
-    }
+.caret {
+  border-left: $caret-width-base solid transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.